Makefile

```diff
@@ -87,3 +87,105 @@ classic-delete:
 
 wf-classic-cluster-conftest: ## Openshift conformance test workflow (Classic ARO using RP)
 	$(MAKE) classic-config-workflow classic-bootstrap-azure classic-create classic-conftest classic-delete classic-clean-bootstrap-azure
+
+###############################################################################
+## Ansible
+##
+REGISTRY ?= registry.access.redhat.com
+TAG ?= $(shell git describe --exact-match 2>/dev/null)
+COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
+ifeq ($(TAG),)
+	VERSION = $(COMMIT)
+else
+	VERSION = $(TAG)
+endif
+
+NO_CACHE ?= true
+PODMAN_REMOTE_ARGS ?=
```

Comment:
How is this populated?

Reply:
These are optional environment variables, Loki's RP-in-containers efforts made use of them so I included them here for completeness